pinot-core/src/main/java/org/apache/pinot/core/operator/dociditerators/SVScanDocIdIterator.java

```diff
@@ -87,7 +94,7 @@ public int next() {
       int limit;
       int batchSize = 0;
       do {
-        limit = Math.min(_numDocs - _nextDocId, OPTIMAL_ITERATOR_BATCH_SIZE);
+        limit = Math.min(_numDocs - _nextDocId, _batch.length);
```

Comment:
(nit)
```suggestion
        limit = Math.min(_numDocs - _nextDocId, _batchSize);
```

Reply:
I've removed `_batchSize` attribute and substitute its usages with `_batchSize.length`